www/src/pages/guidelines/typography.js

```diff
@@ -0,0 +1,600 @@
+
+            <Box maxWidth={{ xl: `40%` }}>
+              <Weight fontFamily="header" fontWeight={1}>
+                Demi
+              </Weight>
+              <Box pb={4}>
+                <code>fontWeights[1]</code> —{` `}
+                <code>{theme.fontWeights[1]}</code>/<code>bold</code>
+                <Text as="p" pt={4}>
+                  Use this for the all headlines but the main page title, and to
+                  emphasize text throughout regular copy.
+                </Text>
+              </Box>
+            </Box>
+            <Box>
+              <Weight fontFamily="header" fontWeight={0}>
+                Normal
+              </Weight>
+              <code>fontWeights[0]</code> — <code>{theme.fontWeights[0]}</code>/
+              <code>normal</code>
+            </Box>
+          </Flex>
+
+          <SectionSubheading pt={6}>Letter Spacing</SectionSubheading>
+
+          <p>
+            Letterspacing (also known as character spacing or tracking) is the
+            adjustment of the horizontal white space between the letters in a
+            block of text.
+          </p>
+
+          <Box pt={4}>
+            <Text as="p" mb={0}>
+              <strong>Normal</strong> — <code>letterSpacings.normal</code> —
+              {` `}
+              <code>{theme.letterSpacings.normal}</code>
+            </Text>
+            <p>Use for almost everything.</p>
+          </Box>
+
+          <hr />
+
+          <Box pt={4}>
+            <Text as="p" mb={0}>
+              <strong>Tight</strong> — <code>letterSpacings.tight</code> —{` `}
+              <code>{theme.letterSpacings.tight}</code>
+            </Text>
+            <p>Use for headlines set in Futura PT.</p>
+            <Heading letterSpacing="tight" lineHeight="solid" pb={8}>
+              Create digital experiences on the edge—faster
+            </Heading>
+          </Box>
+
+          <hr />
+
+          <Box pt={4}>
+            <Text as="p" mb={0}>
+              <strong>Tracked</strong> — <code>letterSpacings.tracked</code> —
+              {` `}
+              <code>{theme.letterSpacings.tracked}</code>
+            </Text>
+            <p>
+              Use for small caps, particularly at small sizes. Lowercase letters
+              don’t need letterspacing, and neither do capital letters (when
+              appear at the beginning of a word, because they’re designed to fit
+              correctly next to lowercase letters). However, when using capital
+              letters together, the default spacing looks too tight.
+            </p>
+            <Text
+              color="grey.60"
+              fontFamily="header"
+              fontSize={1}
+              letterSpacing="tracked"
+              pb={8}
+              css={{
+                textTransform: `uppercase`,
+              }}
+            >
+              greglobinski/
+            </Text>
+          </Box>
+
+          <SectionSubheading pt={6}>Line Heights</SectionSubheading>
```

Comment:
I'm seeing an error in axe for this heading level, which is rendering as an h3. Headings should go in order without skipping levels, so this should either be an h2 or an h2 should be added above it. (This does sort-of seem to be hanging out by itself?)

Reply:
> This does sort-of seem to be hanging out by itself?

:D Yeah, the actual content for this headline was still missing when you reviewed.

I'm guessing this was what axe complained about, as there are two more `<h3>` tags preceding this one ("Letter Spacing" and "Font Weights"). I checked the hierarchy, and this looks OK to me.